include/nanobind/nb_eval.h

```diff
@@ -0,0 +1,63 @@
+/*
+    nanobind/nb_eval.h: Support for evaluating Python expressions and
+                        statements from strings
+
+    Adapted by Nico Schlömer from pybind11's eval.h.
+
+    All rights reserved. Use of this source code is governed by a
+    BSD-style license that can be found in the LICENSE file.
+*/
+
+#pragma once
+
+#include <nanobind/nanobind.h>
+
+#include <utility>
```

Comment:
Are you using anything from `<utility>`?

Reply:
Now that the `std::move`s are gone, not anymore.